crates/bdk/src/wallet/mod.rs

```diff
@@ -388,9 +388,9 @@ impl<D> Wallet<D> {
 
     /// Returns the utxo owned by this wallet corresponding to `outpoint` if it exists in the
     /// wallet's database.
-    pub fn get_utxo(&self, op: OutPoint) -> Option<LocalUtxo> {
+    pub fn get_utxo(&self, op: OutPoint, include_reserved_utxos: bool) -> Option<LocalUtxo> {
```

Comment:
I don't think we should have a `include_reserved_utxos` parameter here... If I want bdk to retrieve a specific outpoint, I usually don't care if it's reserved or not.

I would just call `full_utxos(true)` a couple of lines below and that's it

Reply:
Makes sense.